src/pybind/mgr/dashboard/frontend/src/app/shared/api/settings.service.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { ApiModule } from './api.module';
+
+@Injectable({
+  providedIn: ApiModule
+})
+export class SettingsService {
+  port: number;
```

Comment:
Apparently this properties are not used, and can be removed (`port`, `protocol`and `host`).

Reply:
ack